static/src/javascripts/projects/common/modules/atoms/youtube-player.ts

```diff
@@ -75,71 +82,67 @@ onConsentChange((cmpConsent) => {
 				? Object.values(tcfData.consents).every(Boolean)
 				: false,
 		};
+		resolveInitialConsent('tcfv2');
 	}
 });
 
-interface LocalEvent extends Omit<Event, 'target'> {
-	data: string;
+interface YTPlayerEvent extends Omit<Event, 'target'> {
+	data: number;
 	target: YT.Player;
 }
 
 interface Handlers {
-	onPlayerReady: (event: LocalEvent) => void;
-	onPlayerStateChange: (event: LocalEvent) => void;
+	onPlayerReady: (event: YTPlayerEvent) => void;
+	onPlayerStateChange: (event: YTPlayerEvent) => void;
 }
 
 const onPlayerStateChangeEvent = (
-	event: LocalEvent,
+	event: YTPlayerEvent,
 	handlers?: Handlers,
 	el?: HTMLElement | null,
 ) => {
-	if (el && config.get('page.isDev')) {
-		const states = (window.YT.PlayerState as unknown) as string[];
-		const state = Object.keys(states).find(
-			(key: string) =>
-				states[(key as unknown) as YT.PlayerState] === event.data,
-		);
-		if (state) {
-			console.log(`Player ${el.id} is ${state}`);
+	if (el) {
```

Comment:
Wrap everything that looks for the element into a conditional. There’s no need to run `fastdom` if there’s no element